javascript/javascript1/week2/candy_helper.js

```diff
@@ -0,0 +1,22 @@
+const broughtCandyPrices = [];
+const spendingamount = Math.random() * 100;
+
+function addCandy(candyType, weight) {
+    if (candyType === "Sweet") {
+        candyPrice = 0.5 * weight;
+        broughtCandyPrices.push(candyPrice);
```

Comment:
`candyPrice` is not declared. Declare it before all conditions or push directly to the array
```js
broughtCandyPrices.push(0.5 * weight);
```